control-plane/agents/src/bin/core/tests/policies/mod.rs

```diff
@@ -0,0 +1,61 @@
+#![allow(unused)]
+
+/**
+This file contains tests for any policies that are present
+in the control plane.
+*/
+use deployer_cluster::{Cluster, ClusterBuilder};
+use grpc::operations::{nexus::traits::NexusOperations, node::traits::NodeOperations};
+
+use stor_port::types::v0::transport::{CreateNexus, Filter, Nexus, NexusId};
+
+mod pmodule {
+    include!("../../controller/policies/rebuild_policies.rs");
```

Comment:
why is this being included like this?

Reply:
Apparently, spending couple of hours, it didn't seem feasible to use the types from rebuild_policies.rs in the tests/ by any other means :( . Every other module outside  `tests` is able to see rebuild_policies module, but not `tests`. I'm all ears if there is a possible better way. This is possibly the reason I ended up having to mitigate lot of unused code warnings. The product(non-test) code that's going to use rebuild_policies.rs isn't merged yet.

Reply:
The reason appears to be that the file being included has unused attribute at module level(inner attribute), and it's being imported into a different module so compiler complains about attribute validity. I'll likely wait till the attribute can be removed from included file, i.e. when Abhilash's changes are ready.